back-end/app.js

```diff
@@ -3,13 +3,17 @@ const express = require("express");
 const cors = require("cors");
 const app = express();
 const { PATHS } = require("./config");
-const productsRoutes = require("./routers/productsRoutes");
+
+//-------------------------root router-----------------------------//
+const productsRoute = require("./routers/productsRoutes");
+const loansRoute = require("./routers/loansRoutes");
 
 app.use(cors());
 app.use(express.json());
 app.use(express.static("public"));
 
-app.use(PATHS.products, productsRoutes);
+app.use(PATHS.products, productsRoute);
+app.use(PATHS.loans, loansRoute);
```

Comment:
@puteulanus it's better to have plural form, since it is going to contain multiple routes